new_exploration_programs/segmented_cloud merged_cloud
new_exploration_programs/segmented_cloud source_cloud
new_exploration_programs/matching_pair[] matching_list
